### ISSUE TITLE
Add more tips to the tutorials page

### ISSUE THREAD

**Issue by goobertron** [diaspora/diaspora-project-site#46]
As part of the tutorials series, I have created a 'tips and tricks' page. However there are only two tips on here. For that reason it has not been linked to any other page.

I have uploaded it in case anyone wants to add to these tips. 

Once this has been done, a link will need to be added to the tutorials menu page.

You can find the file [here](https://github.com/goobertron/diaspora-project-site/blob/79514e3a8120ef665a7d62b648cbe0a9a60ce85c/app/views/pages/tips.html.erb) before it is merged.

**Comment by Flaburgan** [comment]
So, the actual URL of the tip page is https://diasporafoundation.org/tips
Do we want to link it from somewhere? From the tutorials page maybe?
Do we have other tips we want to share? Maybe something about the bookmarklet?

**Comment by goobertron** [comment]
> Do we want to link it from somewhere? From the tutorials page maybe?

Yes, if more tips are added and it becomes a more useful page. I'm not convinced it's worth it at the moment.

> Do we have other tips we want to share?

Hopefully!

Diaspora has changed a lot since I created that page – which, in any case, was copied from Kevin Kleinman's original Diasporial site from 2011! So there might well be other tips worth sharing.
